connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java

```diff
@@ -233,7 +234,7 @@ public boolean createTopic(NewTopic topic) {
                     log.debug("Found existing topic '{}' on the brokers at {}", topic, bootstrapServers);
```

Comment:
Since we're in here anyway, can we fix the `if` in the line above to use `cause` instead of `e.getCause()`?

Reply:
Sure.